MODULE.bazel

```diff
@@ -1,6 +1,5 @@
 module(
     name = "rules_img",
-    version = "0.2.5",
```

Comment:
I think we need to keep this version number as it is used to build the release artifacts in Bazel (this code lives in `img/private/release`).

Reply:
Potentially. I'll think about it.